### PR TITLE
Update target-frameworks.md

### PR DESCRIPTION
**Pull request by ArminShoeibi**
Add Unified .NET target frameworks such as .net5.0

### CODE REVIEW

File: docs/reference/target-frameworks.md [hunk 1]
```diff
@@ -85,6 +85,7 @@ Universal Windows Platform | uap | uap [uap10.0] |
 | | | netcoreapp2.2 |
 | | | netcoreapp3.0 |
 | | | netcoreapp3.1 |
+.NET | net | net5.0 |
```
Comment (JonDouglas) [inline docs/reference/target-frameworks.md:88]: This is not quite right. .NET 5 is the successor to netcoreapp3.1(i.e. net5.0 for short). Thus we should mimic the table found here: https://docs.microsoft.com/en-us/dotnet/standard/frameworks#supported-target-frameworks

Here's a tool to show the precedence for reference!

https://nugettoolsdev.azurewebsites.net/5.8.0/framework-precedence?framework=net5.0&exludedIdentifiers=

If you'd like to update your PR to reflect that, we can merge! Thanks in advance!

**Review by JonDouglas — CHANGES_REQUESTED** [review]
After quick changes, we can merge!

File: docs/reference/target-frameworks.md (hunk 1 above)
Comment (ArminShoeibi) [reply docs/reference/target-frameworks.md]: Thank you, Done.
https://github.com/NuGet/docs.microsoft.com-nuget/pull/2398/commits/ff1ef787a30a368bebb3407ac7f0711340cd2094

File: docs/reference/target-frameworks.md [hunk 2]
```diff
@@ -78,13 +78,15 @@ Universal Windows Platform | uap | uap [uap10.0] |
...
-.NET Core App | netcoreapp | netcoreapp1.0 |
+.NET 5+ (and .NET Core) | netcoreapp | netcoreapp1.0 |
 | | | netcoreapp1.1 |
 | | | netcoreapp2.0 |
 | | | netcoreapp2.1 |
 | | | netcoreapp2.2 |
 | | | netcoreapp3.0 |
 | | | netcoreapp3.1 |
+| | net | net5.0 |
+| | net | net6.0 |
```
Comment (JonDouglas) [inline docs/reference/target-frameworks.md:89]: remove this `net` to be consistent & we'll merge!

Comment (JonDouglas) [reply docs/reference/target-frameworks.md]: Sorry just one of them! 

image

Comment (JonDouglas) [reply docs/reference/target-frameworks.md]: you can keep the `net` on `net5.0`! It looks great there.

Comment (ArminShoeibi) [reply docs/reference/target-frameworks.md]: > you can keep the `net` on `net5.0`! It looks great there.
agreed.

**Review by JonDouglas — APPROVED** [review]
Thank you so much for the contribution! It's really appreciated!

**Comment by ArminShoeibi** [comment]
> Thank you so much for the contribution! It's really appreciated!

I'm very happy this is my very first contribution to OSS.

**Comment by JonDouglas** [comment]
Woohoo! Congrats @ArminShoeibi

**Comment by loic-sharma** [comment]
Excellent first contribution!

test

**Comment by ArminShoeibi** [comment]
> Woohoo! Congrats @ArminShoeibi
🤗
Thank you.

**Comment by ArminShoeibi** [comment]
> Excellent first contribution!
> 
> test

Thank you very much 🌹

**Pull request merged**